com.github.petebuffon.yafc.yml

```diff
@@ -0,0 +1,25 @@
+app-id: com.github.petebuffon.yafc
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: YAFC
+finish-args:
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--socket=fallback-x11"
+  - "--share=ipc"
+  - "--filesystem=~/.factorio/mods:ro"
+  - "--filesystem=~/.steam/steam/steamapps/common/Factorio:ro"
```

Comment:
This misses flatpaked Steam.

Reply:
Added - "--filesystem=xdg-run/app/com.valvesoftware.Steam:ro